### PR TITLE
feat: setup tailwind

### PR DESCRIPTION
**Pull request by alexnm**

### CODE REVIEW

**Review by alexnm — COMMENTED** [review]
Left a few comments

File: src/layouts/BaseLayout.astro [hunk 1]
```diff
@@ -83,7 +83,6 @@ const description = `JSHeroes is an yearly event organized by the local JS commu
         --color-neutral-light: #ffffff;
         --color-neutral-separator: #ebe7e7;
         --color-primary: #0b1b44;
-        --color-logo: #0098ff;
         --color-accent-pink: #ff6078;
         --color-accent-orange: #ffcc67;
         --color-accent-green: #65d3bb;
```
[the remark this answers is not in the document]
Comment (AlexandruLupu) [reply src/layouts/BaseLayout.astro:88]: Yes, until we remove all plain css classes that use them:
image

File: src/layouts/BaseLayout.astro [hunk 2]
```diff
@@ -132,70 +131,85 @@ const description = `JSHeroes is an yearly event organized by the local JS commu
...
+        font-weight: theme("fontWeight.semibold");
+      }
+
+      h3 {
+        font-size: theme("fontSize.base");
+        font-weight: theme("fontWeight.semibold");
+      }
+
+      p {
+        margin: theme("spacing.4") 0;
```
[the remark this answers is not in the document]
Comment (AlexandruLupu) [reply src/layouts/BaseLayout.astro:149]: It is part of the tailwind typography addon, but the `prose` class was added just for the pages that render markdown content. Once we remove the plain css, we can fully use the addon.

File: src/pages/media.astro [hunk 3]
```diff
@@ -357,6 +357,7 @@ import Section from "../components/Section.astro";
   .gallery {
```
[the remark this answers is not in the document]
Comment (AlexandruLupu) [reply src/pages/media.astro]: Removed, thanks!

**Pull request merged**